test/com/google/enterprise/adaptor/DocIdSenderTest.java

```diff
@@ -51,6 +50,15 @@
   private ExceptionHandler runtimeExceptionHandler
       = new RuntimeExceptionExceptionHandler();
 
+  private static final Map<GroupPrincipal, Collection<Principal>> SAMPLE_DATA
```

Comment:
I don't like having the test data buried where it is. Maybe at least add a comment here about the contents of each of these?